test/test_performance.py.in

```diff
@@ -65,6 +69,17 @@ def _raw_to_png(csv_data, png_path):
 
     plt.savefig(png_path)
 
+    dataframe[['T_experiment', 'cpu_usage (%)']].plot(x='T_experiment')
+
+    plt.title('@TEST_NAME@ CPU usage (%) Test')
+
+    plt.savefig(png_path[:-4] + "_cpu_usage.png")
+
+    dataframe.plot(kind='bar',y=['received', 'sent', 'lost'])
+
+    plt.title('@TEST_NAME@  Received/Sent/Lost packages')
```

Comment:
Is "packages" the right term, or "packets"? Actually, maybe "messages" is more precise.